set.mm

```diff
@@ -467526,6 +468136,94 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
       CGHZCIHZDGHZDIHZBCJKHZUCALHSTCMHCNOUAUBDMHDNOUCPABCDEFQR $.
   $}
 
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Division Ring Extensions
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    drgext.b $e |- B = ( ( subringAlg ` E ) ` U ) $.
+    drgext.f $e |- F = ( E |`s U ) $.
+    drgext.1 $e |- ( ph -> E e. DivRing ) $.
+    drgext.2 $e |- ( ph -> F e. DivRing ) $.
+    drgext.3 $e |- ( ph -> U e. ( SubRing ` E ) ) $.
+    $( The additive neutral element of a division ring extension.  (Contributed
```

Comment:
Are the hypotheses drgext.f and drgext.2 used? I see no `F` in the following theorems.

Reply:
They are not used for the first 3 theorems `~drgext0g`, `~drgextvsca` and `~drgext0gsca`, but the fact that `F` is a division ring is used the the later theorems. I'll move those two hypotheses later so that they only appear when needed.

Reply:
OK,  `F` seems to be an abbreviation for ``( E |`s U )``  used within the proofs, but still does not appear in the theorems themselves. So the hypothesis ``drgext.3 $e |- ( ph ->  ( E |`s U )  e. DivRing ) $.`` should be sufficient.

I do not know if there is any (unwritten) convention to define symbols in hypotheses only if they are used in the theorems (and not in proofs only), and I am also not aware of any similar case in set.mm.

Nevertheless, this is OK for now.

Reply:
I also wanted to follow the same model as for the main theorem, `fedgmul`. 
We'll see if I maybe reword those theorems using the proposed new syntax (maybe a "Division Ring Extension syntax").